### ISSUE TITLE
Cannot ]add PEtab

### ISSUE THREAD

**Issue by MarcusGalea** [sebapersson/PEtab.jl#201]
I have no issue when forking and running your code, but for some reason, I cannot add your package into an empty environment.
I get the following error:
```
ERROR: LoadError: MethodError: no method matching PEtab.PEtabModel(::String; verbose::Bool, build_julia_files::Bool, write_to_file::Bool)
```
Is there something I can do to fix this?

**Comment by sebapersson** [comment]
Thanks for reporting! 

This is really strange, never seen this before.  Which version of Julia are you on, and which OS? Also when adding the package can you try to explicetely add the latest version?

**Comment by sebapersson** [comment]
Thanks for the additional info.

This is good news if you are using PEtab.jl with Catalyst.jl, because then the problem is likely in the SBML import (which is only for models in the PEtab standard format).

The issue is likely due to an update in the internals of SBMLImporter.jl.  I think you can fix this by in the Project.toml file for the environment add
```
[deps]
SBMLImporter = SBMLImporter = "210efffb-c3c8-456d-a807-6f55560b12fe"

[compat]
SBMLImporter = "=1.1.2"
```

This pre-compilation issue should be resolved when PEtab.jl updates to Catalyst v14 (and in extension SBMLImporter v2). I am to a large degree done with updating PEtab, so within two weeks this should be resolved. But before then, hopefully this workaround solves the problem!

**Comment by sebapersson** [comment]
This is strange, and I cannot reproduce this locally (on Linux).

Does it work using this exact Project.toml file:

```
[deps]
PEtab = "48d54b35-e43e-4a66-a5a1-dde6b987cf69"
SBMLImporter = "210efffb-c3c8-456d-a807-6f55560b12fe"

[compat]
SBMLImporter = "=1.1.2"
```

Otherwise, I will try to find a Windows computer and see if I can reproduce the issue. Hopefully this should also not be a problem when PEtab.jl is `updated.

**Comment by MarcusGalea** [comment]
Yes, even after instantiating with that exact Project.toml file, i get the same error. I'm looking forward to the fix, I'm really excited about using this package.

**Comment by sebapersson** [comment]
I have just released PEtab version 2.15, which I think should solve the above pre-compilation issue . Please check if the new version resolves the problem.
